lib/promiscuous/backend.rb

```diff
@@ -0,0 +1,50 @@
+module Promiscuous::Backend
+  extend Promiscuous::Autoload
+  autoload :Bunny, :Poseidon, :Null, :Fake, :File
+
+  class << self
+    attr_accessor :driver
+    attr_accessor :driver_class
+    attr_accessor :subscriber_worker_module
+
+    def driver=(value)
+      disconnect
+      @driver_class = value.nil? ? nil : "Promiscuous::Backend::#{value.to_s.camelize.gsub(/backend/, 'Backend')}".constantize
```

Comment:
you shouldn't have to specify namespace as its in the same namespace. also that gsub is dirty. remove backend completely.